website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -136,10 +136,6 @@ security fixes.
 
 | `consul-dataplane` Version Range | Associated Consul Enterprise version     | Contained Envoy Binary Version |
 | -------------------------------- | ---------------------------------------- | ------------------------------ |
-| 1.5.0 - 1.5.latest               | 1.18.x Ent                               | Envoy 1.29.x                   |
-| 1.4.0 - 1.4.latest               | 1.18.x Ent                               | Envoy 1.28.x                   |
```

Comment:
I've removed these as neither dataplane release spans multiple major Envoy versions, which is what this section of the doc is meant to be